src/util.ts

```diff
@@ -2,13 +2,12 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-
-import { reporter } from './telemetry/reporter';
+/* tslint:disable */
```

Comment:
same here.

Reply:
This change should be reverted. It's a mistake.

Reply:
:+1:  Please double check as I saw it in a few places.